src/integrationtest/python/steps/step_create_account.py

```diff
@@ -0,0 +1,11 @@
+from PyQt5.QtWidgets import QDialog
+
+from PyQt5.QtTest import QTest
+from PyQt5.QtCore import Qt
+
+from behave import use_step_matcher, when, then
+
+from test.steps.step_helpers import assert_equal, assert_is_not, assert_is
```

Comment:
this looks to make stuff fail:

```
  File "src/integrationtest/python/steps/step_create_account.py", line 8, in <module>
    from test.steps.step_helpers import assert_equal, assert_is_not, assert_is
ModuleNotFoundError: No module named 'test.steps'
```

changing it to `src.integrationtest.python.steps` works for me